Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM coco/k8s-cli-utils
+FROM coco/k8s-cli-utils:kube-aws-upgrade
```

Comment:
Can you update this to a stable version?

Reply:
Yep, after I merge https://github.com/Financial-Times/k8s-cli-utils/pull/4 :)